src/server/graphql/httpGraphQLHandler.js

```diff
@@ -4,10 +4,10 @@ import {graphql} from 'graphql';
 export default async(req, res) => {
   // eslint-disable-next-line global-require
   const Schema = require('./rootSchema');
-  const {query, variables, ...newContext} = req.body;
```

Comment:
now that our context has things like `exchange` and `socket`, i don't want some joker throwing in one of those to mess with us & cause the server to throw an error. no real risk, just avoiding a possible annoyance.